devtools/common/src/main/resources/templates/dockerfile-jvm.ftl

```diff
@@ -0,0 +1,19 @@
+####
+# Before building the docker image run:
```

Comment:
Maybe add a line (before) explaining that this run the application in JVM mode. I know it's obvious, but well... you know ;-)

Reply:
Sure thing!

Reply:
Done